src/analysis/evaluation.py

```diff
@@ -190,9 +227,16 @@ def monthly_score(
         )
         model_files[model] = xr.open_dataset(pred_path).isel(time=0)
 
-    true_data = xr.open_dataset(
-        data_path / f"features/{experiment}/test" f"/{pred_year}_{month}/y.nc"
-    ).isel(time=0)
+    if true_data_experiment is None:
+        true_data_path = (
+            data_path / f"features/{experiment}/test" f"/{pred_year}_{month}/y.nc"
+        )
+    else:
```

Comment:
When would `true_data_experiment` be different from `experiment`?

Reply:
when the model file has been renamed (when running these experiments I have to rename the model directory `one_month_forecast_EXPT1`)but the underlying `features/one_month_forecast` data remains the same